raiden-cli/raspberrypi.md

```diff
@@ -0,0 +1,112 @@
+# Procedure to run raiden-cli on the Raspberry Pi
+
+This tutorial assumes the following:
+
+1. These steps have to be run on the _Raspberry Pi 4_.
+2. _Raspberry Pi_ has to be in the same network as your **PC** with SSH enabled.
+3. The _Raspberry Pi_ is installed with the [Raspberry Pi OS Lite](https://www.raspberrypi.org/software/operating-systems/).
+
+The steps to copy files from your **PC** to the **pi** have been explicitly mentioned and have to be run on your **PC**.
+
+```sh
+sudo apt update
+
+sudo apt full-upgrade
+
+sudo apt install git
+```
+
+Perform a [git install](https://github.com/nvm-sh/nvm#git-install) of the NVM
+
+```sh
+git clone https://github.com/nvm-sh/nvm.git .nvm
+
+cd ~/.nvm
+
+git checkout v0.38.0
+
+. ./nvm.sh
+
+nvm install --lts
+
+npm install --global yarn
+```
+
+Check the arm system
+
+```sh
+uname -m
+armv7l
```

Comment:
this sounds like a command to run, please write some text on what the output should be

Reply:
done